app/src/main/java/swati4star/createpdf/fragment/AddImagesFragment.java

```diff
@@ -288,16 +286,16 @@ private void setTextAndActivateButtons(String path) {
 
     @Override
     public void onPopulate(ArrayList<String> paths) {
-        populateUtil(mActivity, paths, this, mLayout, mLottieProgress, mRecyclerViewFiles);
+        getInstance().populateUtil(mActivity, paths, this, mLayout, mLottieProgress, mRecyclerViewFiles);
```

Comment:
hre also, use `CommonCodeUtils.getInstance`

Reply:
@Swati4star made the changes